helmfile.d/upstream/grafana/grafana/templates/pvc.yaml

```diff
@@ -25,6 +25,9 @@ spec:
   resources:
     requests:
       storage: {{ .Values.persistence.size | quote }}
+  {{- if (lookup "v1" "PersistentVolumeClaim" (include "grafana.namespace" .) (include "grafana.fullname" .)) }}
```

Comment:
I didn't realize this was added in this chart version, this might get problematic once we upgrade with [this set to false](https://github.com/elastisys/compliantkubernetes-apps/pull/2245/files#diff-c3cce006b6a3a81d50da78678bf67fb929ab7873be14383792802e2e30bd5379R22), see my comment in the PR [here](https://github.com/elastisys/compliantkubernetes-apps/pull/2245#issuecomment-2343073741) and the linked issues. I guess this will be the same for apps v0.39.1. Not sure how to approach this, maybe we need to change `lookupVolumeName` to true on main so that it does not break in future upgrades and live with the false-diffs? What do others think

Reply:
This will be resolved in apps v0.42, so non-blocking for this PR